Content.Client/Corvax/JoinQueue/QueueState.cs

```diff
@@ -0,0 +1,55 @@
+using Content.Shared.Corvax.JoinQueue;
+using Robust.Client.Audio;
+using Robust.Client.Console;
+using Robust.Client.State;
+using Robust.Client.UserInterface;
+using Robust.Shared.Player;
+
+namespace Content.Client.Corvax.JoinQueue;
+
+public sealed class QueueState : State
+{
+    [Dependency] private readonly IUserInterfaceManager _userInterface = default!;
+    [Dependency] private readonly IClientConsoleHost _console = default!;
+
+
+    // TODO Make this the admin new player notification sound or something
+    private const string JoinSoundPath = "/Audio/Effects/voteding.ogg";
```

Comment:
This should be done

Reply:
I was gonna do it when I added the admin new player notifications

Reply:
https://github.com/Simple-Station/Einstein-Engines/pull/318